mucgpt-frontend/src/components/AssistantDialogs/shared/sections/ConversationOptionsSection.module.css

```
@@ -0,0 +1,353 @@
+    gap: 8px;
+    flex-direction: column;
+    justify-content: center;
+    min-width: 0;
+}
+
+.dynamicFieldInput {
+    flex: 1;
+    min-width: 0;
+}
+
+.dynamicFieldInput:global(.fui-Input),
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)